packages/client/lib/net/protocol/snapprotocol.ts

```diff
@@ -1,7 +1,7 @@
 import {
   accountBodyFromSlim,
   accountBodyToSlim,
-  bigIntToBuffer,
+  devP2PBigIntToBuffer as bigIntToBuffer,
```

Comment:
See my other comment, we can use the existing `bigIntToUnpaddedBuffer` function for these cases.

I would also suggest to *not* rename but to use plainly, then it gets more explicit what conversion takes place here, renaming could easily mislead to think that plain `bigIntToBuffer` is used, which is also an existing function in `Util`.